src/services/azblob/backend.rs

```diff
@@ -303,8 +304,12 @@ impl Accessor for Backend {
     }
 
     async fn write(&self, path: &str, args: OpWrite, r: BytesReader) -> Result<u64> {
-        let mut req =
-            self.azblob_put_blob_request(path, Some(args.size()), AsyncBody::Reader(r))?;
+        let mut req = self.azblob_put_blob_request(
+            path,
+            Some(args.size()),
+            Some(args.content_type()),
```

Comment:
As I said, it's wrong. Please only set content type when it's some. Using `Option<String>` here so that we know it's provided by user.

Reply:
If the user does not provide or just does provide an empty `content-type` string, the content-type will be set to an empty string, and the final HTTP request will not include this header.

Reply:
For services that don't support content_type, we need a way to distinguish `None` and `Empty` value.